packages/rollup-tests/src/ignored-by-unsupported-features.md

```diff
@@ -274,6 +280,24 @@
 ### The rolldown `output.dir` default to be `dist`, the rollup not specific `dir` or `file` by default
  - rollup@hooks@Throws when not specifying "file" or "dir"
 
+### object `options.globals` is broken
```

Comment:
This one is really strange. If I run `rollup@form@arbitrary-module-namespace-identifiers` with `solo: true`, no error happens. But if I run without `solo: true`, I get the following error:

> Error: Value is non of these types `HashMap`, `ThreadsafeFunction`,  on BindingOutputOptions.globals on BindingBundlerOptions.outputOptions

The value passed to `globals` is `{ external: 'external' }`, so it should be able to be deserialized to `HashMap`.